### DISCUSSION TITLE
What happened with the launch files

### DISCUSSION THREAD

**Discussion by FPSychotic** [realsenseai/realsense-ros#1847] [General]
What happened with the launch files, they are plenty of -1, what thats mean?
Why the imu is disabled and if it is activated  result in a sinchronization error at launching?
It doesn't take the .json file when I specify the address.
Did I  miss something? In the launch uses or similar?
New values to setup but I cannot find what they mean

**Comment by doronhi** [comment]
Could you elaborate what version do you use? ROS1 or ROS2? Even more details regarding the version will be better.
The -1 values mean for the wrapper to choose the default values, depending on the type of sensor and connection you use. It is meant to behave the same way as **realsense-viewer** chooses the default profile for each sensor.
The IMU is disabled by default because most users are not using it so there is no point in loading the systems. You can enable it by adding `enable_gyro:=true enable_accel:=true`. I am not aware of a "sinchronization error at launching".  If there is one, it would help if you open an issue and describe it.
The same addvice goes for the .json file: could you open an issue and add the command line you use, the output and if possible, the .json file itself?
Regarding the "New values to setup" - if you can elaborate and they are missing from the README.md file, I will be happy to rectify.
Thanks.

**Reply by FPSychotic** [reply]
Ok just it was weird for and wanted general information before open a issue. 
Is there some place were new features and fixed errors are explained. There is new things, filters,.options, that I don't know what they mean, or behave or when use them.
Thanks by answer do fast.
Would be great to know if the wheel odometry input is finally working in t265.
You are a busy person, and dont want ask by everything.
Cheers

**Reply by doronhi** [reply]
Hi @FPSychotic , I somehow missed this conversation for a very long while and I apologize for that.
We describe the parameters and features in the README.md file, as this place is available and stable.
So far Requests were brought up as issues. There hasn't been a place to discuss new features but maybe it's a good idea.
I released a new [beta version](https://github.com/IntelRealSense/realsense-ros/releases/tag/v4.0.0-beta) and opened a [discussion](https://github.com/IntelRealSense/realsense-ros/discussions/2167) so there will be a place for discussing its usefulness and consider the new interfaces. Maybe it's a good idea to start a discussion for every new release.

**Reply by FPSychotic** [reply]
Yeah would be nice every release have open a conversation , that would make comunity and can receive opinions and help from others, that even would help to devs to receive less issues and maybe even help to decide what improvements people would like
